python/lsst/faro/base/BaseSubTasks.py

```diff
@@ -21,31 +21,45 @@
 
 from lsst.afw.table import SourceCatalog
 from lsst.pipe.base import Struct, Task
-from lsst.pex.config import Config, Field
+from lsst.pex.config import Field, DictField
 from lsst.verify import Measurement
 
 from lsst.faro.utils.matcher import mergeCatalogs
 from lsst.faro.utils.calibrated_catalog import CalibratedCatalog
-
+from lsst.faro.base.ConfigBase import MeasurementTaskConfig
 import astropy.units as u
 import numpy as np
 from typing import Dict, List
 
 __all__ = (
     "NumSourcesTask",
+    "NumSourcesConfig",
     "NumSourcesMatchedTask",
     "NumSourcesMergeTask",
     "NumpySummaryConfig",
     "NumpySummaryTask",
 )
 
 
-class NumSourcesConfig(Config):
+class NumSourcesConfig(MeasurementTaskConfig):
```

Comment:
I think ideally we would use selectWhenTrue here, except that this is also the default function for the analysis contexts that use FITS inputs.